### PR TITLE
Create 2022-november-seminar.tweet

### PR DESCRIPTION
**Pull request by lucaferranti**

### CODE REVIEW

File: tweets/2022-november-seminar.tweet [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Ever wondered how to make #parallel #programming and working with #supercomputers more productive?
+
+At Nordic-RSE seminar series today Brad Chamberlain from HPE will tell us how @ChapelLang can help solving those issues.
+
+Talk at 16:00 CET, free for everyone to join.
```
Comment (lucaferranti) [inline tweets/2022-november-seminar.tweet:5]:
```suggestion
Talk at 16:00 CET, everyone welcome
```

File: tweets/2022-november-seminar.tweet [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Ever wondered how to make #parallel #programming and working with #supercomputers more productive?
```
Comment (lucaferranti) [inline tweets/2022-november-seminar.tweet:1]:
```suggestion
Does #parallel #programming and working with #supercomputers feel difficult?
```

File: tweets/2022-november-seminar.tweet [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+At Nordic-RSE seminar series today Brad Chamberlain from HPE will tell us how @ChapelLang can help solving those issues.
```
Comment (lucaferranti) [inline tweets/2022-november-seminar.tweet:3]:
```suggestion
At Nordic-RSE seminar today, Brad Chamberlain from HPE will tell us how @ChapelLang can help you solving this issues.
```

**Review by rantahar — APPROVED** [review]

**Review by mtomasini — COMMENTED** [review]
Looks good to me! I just wonder whether #parallel and #programming should be two different hashtags, since parallel programming is a specific topic.

**Pull request merged**